# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uucp/uucp-owner.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./contrib/uupoll.shar.orig	2003-07-09 12:53:50.000000000 -0600
+++ ./contrib/uupoll.shar	2003-07-09 12:54:55.000000000 -0600
@@ -37,7 +37,7 @@
 X
 # The user name/group that should own the resulting executables.
 # Both should run suid.
-owner = uucp.daemon
+owner = uucp.uucp
 X
 # Which mode should the resulting executables have.
 emode = 4111
